package.json

```diff
@@ -14,8 +14,9 @@
     "start": "node built/app.server/server.js"
   },
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
Note all the existing dependencies are `--exact` but all the new ones have been added with a `^` which is non-deterministic. Can we make them exact like the existing.

Reply:
done